src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -81,10 +81,6 @@
             <tag name="kernel.event_subscriber" />
         </service>
 
-        <service id="resolve_controller_name_subscriber" class="Symfony\Bundle\FrameworkBundle\EventListener\ResolveControllerNameSubscriber">
```

Comment:
This service should have been deprecated in 4.1, it should be in 4.4.

Reply:
I think this PR should also remove the `ControllerNameParser` class as well as it is part of the same deprecated feature which is the `a:b:c` notation, and the corresponding `controller_name_converter` should be deprecated since 4.1 as well.

Reply:
To me all deprecations from #26085 should be removed in one PR to be self-contained. I can work on this.

Reply:
Don't worry I will have time to finish this ;)

Reply:
done